### DISCUSSION TITLE
TryOut

### DISCUSSION THREAD

**Discussion by HenkMutsaerts** [OSIPI/osipi.github.io#158] [General]
Who can read this and who can see this?

**Comment by ltorres6** [comment]
I can see this!
---
Luis Torres
Scientific Solutions Engineer | Flywheel
m: 505-730-0392
***@***.***
flywheel.io
<https://flywheel.io/?utm_source=signature&utm_medium=email&utm_campaign=2021newsignature>
<https://flywheel.io/?utm_source=signature&utm_medium=email&utm_campaign=2021newsignature>

**Comment by lauracbell** [comment]
I can read this too!(And apologies if this is a long d-list!)On Mar 21, 2023, at 12:38 PM, Luis Torres ***@***.***> wrote:﻿
I can see this!
---
Luis Torres
Scientific Solutions Engineer | Flywheel
m: 505-730-0392
***@***.***
flywheel.io
<https://flywheel.io/?utm_source=signature&utm_medium=email&utm_campaign=2021newsignature>
<https://flywheel.io/?utm_source=signature&utm_medium=email&utm_campaign=2021newsignature>

**Comment by plaresmedima** [comment]
And me!!

Prof. Steven Sourbron, PhD
Chair in Medical Imaging Physics
University of Sheffield, UK
*https
<https://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron>://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron
<https://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron>*
*Sheffield abdominal imaging research page*
<https://www.sheffield.ac.uk/medicine/research/research-themes/medical-imaging/medical-imaging-research/abdominal-imaging>
*Open source initiative in perfusion imaging* <http://www.osipi.org>
*Magnetic resonance imaging biomarkers for chronic kidney disease
<https://renalmri.org/>*
